lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2729,10 +2729,13 @@ FIRModuleParser::parsePortList(SmallVectorImpl<PortInfoAndLoc> &result,
         parseOptionalInfo(info))
       return failure();
 
+    ArrayAttr annotations = ArrayAttr({});
+    getAnnotations(getModuleTarget() + ">" + name.getValue(), annotations);
```

Comment:
Wondering if we could encapsulate the format into a wrapper API, that returns the appropriate target string, given the port name, something like, ``getPortAnnotationTargetName(getModuleTarget(), name.getValue())``. 
It could be reused at all  the other occurrences (`` __ + ">" + __ ``) in the file . 
But maybe your current implementation is more efficient.